### PR TITLE
Feature Styling

### PR DESCRIPTION
**Pull request by taldr27**
### BookStore: Added some styles corrections.
In this milestone of the BookStore project I:
- Added styles following the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378).
- Added styles corrections to the navBar.
- Added inline styles to some of the progression chapters.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -31,14 +31,20 @@ const Book = (
...
                     </div>
                   </div>
+                  <div className="line" />
                   <div className="current-chapter">
-                    <span>Current Chapter</span>
-                    <span>Chapter 20</span>
-                    <button type="button">Update progress</button>
+                    <span style={{ fontSize: '0.813rem', opacity: '0.5' }}>Current Chapter</span>
+                    <span>Chapter 3: &ldquo;A Lesson Learned&ldquo;</span>
+                    <button style={{ marginTop: '1em', width: '70%' }} type="button">Update progress</button>
```
Comment (danmainah) [inline src/components/Book.js]: - [x] @taldr27 great work in your styling however it's bad practice to use inline styling in your components, instead, I would recommend you create a variable for your style and then pass it to react style eg
```
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){

    return <div style={myComponentStyle}> Inline Styled Component</div>

}

```

File: src/pages/Categories.js [hunk 2]
```diff
@@ -20,3 +18,22 @@ const Categories = () => {
...
+//   .then((response) => response.text())
+//   .then((json) => console.log(json));
+
+// createAsyncThunk('book/fetchBooks', () => axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/EqZ6YUUbJz8PKUL0OiMU/books/')
+//   .then((response) => response.text())
+//   .then((json) => console.log(json)));
+//   fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/EqZ6YUUbJz8PKUL0OiMU/books/', {
+//   })
+//     .then((response) => response.json())
+//     .then((json) => console.log(json));
```
Comment (danmainah) [inline src/pages/Categories.js]: - [x] Kindly remove the commented code, its bad practise to have commented code in your codebase.

File: src/components/Book.js (hunk 1 above)
Comment (danmainah) [reply src/components/Book.js]: - [x] @taldr27 great work in your styling however this section color doesn't match the zeplin one given, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @taldr27 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Reem-lab — APPROVED** [review]
# Hi @taldr27, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights
- No Linters error ✅
- Followed Git flow ✅
- Great working on the design  ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**